CHANGELOG.md

```diff
@@ -6,6 +6,12 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 
 ## [Unreleased]
 
+### Added
+- arm64 image build support
+
+### Changed
+- remove usage of syscall() to support non-amd64 systems
```

Comment:
@quincycheng some comments on this:

1. Does this PR have an issue that it resolves? If not, please create one. In any case, please add a link to it to align with the standard (you can see examples below).
2. Please capitalized the first words of the sentence.